src/NuGet.Core/NuGet.Protocol/SourceRepositoryProvider.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
This looks like a BOM removal. Or addition. Not sure.

Reply:
Indeed. GitHub doesn't show it, but `git diff` on the command line shows it clearly. Our `.editorconfig` defines `charset = utf-8`, which is the no BOM charset.